mmv1/templates/terraform/examples/bigtable_app_profile_row_affinity.tf.tmpl

```diff
@@ -0,0 +1,37 @@
+resource "google_bigtable_instance" "instance" {
+  name = "{{index $.Vars "instance_name"}}"
+  cluster {
+    cluster_id   = "cluster-1"
+    zone         = "us-central1-a"
+    num_nodes    = 3
+    storage_type = "HDD"
+  }
+  cluster {
+    cluster_id   = "cluster-2"
+    zone         = "us-central1-b"
+    num_nodes    = 3
+    storage_type = "HDD"
+  }
+  cluster {
+    cluster_id   = "cluster-3"
+    zone         = "us-central1-c"
+    num_nodes    = 3
+    storage_type = "HDD"
+  }
+
+  deletion_protection  = "{{index $.Vars "deletion_protection"}}"
+}
+
+resource "google_bigtable_app_profile" "ap" {
+  instance       = google_bigtable_instance.instance.name
+  app_profile_id = "{{index $.Vars "app_profile_name"}}"
+
+  // Requests will be routed to the following 2 clusters.
+  multi_cluster_routing_use_any = true
+  multi_cluster_routing_cluster_ids = ["cluster-1", "cluster-2"]
+
+  row_affinity = true
+
+  ignore_warnings               = true
```

Comment:
something happened to the format here...

Reply:
yeah weird...it may have happened because I was messing around with the `gofmt` command